src/Factories/TestCaseFactory.php

```diff
@@ -35,6 +35,15 @@ final class TestCaseFactory
         Annotations\Groups::class,
     ];
 
+    /**
+     * The list of annotations.
+     *
+     * @var array<int, class-string>
```

Comment:
Can probably enforce that it's a class string of type `Attributes\Attribute` 🤷🏻
```suggestion
     * @var array<int, class-string<\Pest\Factories\Attributes\Attribute>>
```

Reply:
Didn't know this was possible, thank you!